pkg/projects/dir.props

```diff
@@ -108,6 +108,20 @@
     </BuildRID>
   </ItemGroup>
 
+  <!-- RIDs to remove from the runtime dependencies, so the lineup packages don't have distro specific dependencies.
+       Instead, the linux-x64 runtime will be used on all linux distros by default. 
+       NOTE: There isn't a linux-arm build yet, so we can't remove any arm RIDs. -->
+  <ItemGroup>
```

Comment:
Should we make this inclusive instead of exclusive? That way we don't have anything creep in that we don't want.